src/github.rs

```diff
@@ -500,11 +614,130 @@ pub struct IssuesEvent {
     pub label: Option<Label>,
 }
 
+#[derive(Debug, serde::Deserialize)]
+pub struct IssueSearchResult {
+    pub total_count: usize,
+    pub incomplete_results: bool,
+    pub items: Vec<Issue>,
+}
+
 #[derive(Debug, serde::Deserialize)]
 pub struct Repository {
     pub full_name: String,
 }
 
+impl Repository {
+    fn base_url(&self) -> &str {
```

Comment:
This is not the base_url for a repository (and also feels a bit odd? maybe just inline)

Reply:
Done